integration-tests/spring-boot-properties/README.md

```diff
@@ -0,0 +1,12 @@
+### Current differences with Spring Boot properties.
+
+1. Default values assigned with `=` operator are not supported.
+2. Getters and setters are needed for all fields, including nested classes.
+3. `@Bean` needs to be removed from a method which is annotated with `@ConfigurationProperties`.
+Otherwise two beans will be created and injection will fail.
+4. `@NestedConfigurationProperty` can be used but is not required.
+Nested property classes work out of the box with inner and outer classes.
+5. `@ConstructorBinding` is not supported.
```

Comment:
This would be a nice follow up that would also work for Quarkus's `@ConfigProperties`.